.github/workflows/nouse_install.yml

```diff
@@ -100,8 +100,8 @@ jobs:
     strategy:
       matrix:
         # https://github.com/actions/runner-images/blob/main/images/macos/macos-11-Readme.md
-        # https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md
-        os: [macos-12]
+        # https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md
```

Comment:
We don't need the macos-11-Readme.md link.  Let's remove it in the future.

Reply:
sure